File: backup/Invest-Invest/Invest-Invest-Repository/src/main/java/com/illunex/invest/invest/entity/ExamItemScore.java
package com.illunex.invest.invest.entity;

import javax.persistence.*;

@Entity
public class ExamItemScore {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long examItemScoreIdx;
    String score;

    @OneToOne
    ExamItem examItem;
}

File: microservices/invest/src/main/kotlin/com/illunex/invest/invest/persistence/entity/ExamItemScore.kt
package com.illunex.invest.invest.persistence.entity

import javax.persistence.*

@Entity
class ExamItemScore {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var examItemScoreIdx: Long? = null
    var score: String? = null

    @OneToOne
    var examItemEntity: ExamItemEntity? = null
}
